internal/jennies/java/templates/types/class.tmpl

```diff
@@ -1,7 +1,11 @@
 package {{ .Package }};
+{{- $class := include "class" . }}
```

Comment:
I don't understand this change.

Reply:
That's because rendering this partial template can trigger the addition of new imports. To be able to include these in the final rendered template, I render this partial into a variable before the imports themselves are rendered.